classes/Renderer/Markdown.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace MySociety\TheyWorkForYou\Renderer;
+
+/**
+ * Markdown Renderer
+ *
+ * Use for converting markdown into help pages.
+ */
+
+class Markdown
+{
+    public function markdown_document($this_page){
+        // This function takes a markdown file and converts it to HTML
+    
+        $markdown_file = '../../../markdown/' . $this_page . '.md';
+        $Parsedown = new \Parsedown();
+        
+        $text = file_get_contents($markdown_file);
+        $html = $text;
+        $html = $Parsedown->text($text);
+        
+        # title is the first h1
+        preg_match('/<h1>([^<]+)<\/h1>/i', $html, $matches);
+        
+        $title = $matches[1];
+        
+        $html = preg_replace_callback('/<h([1-3])>([^<]+)<\/h[1-3]>/i', function($matches) {
+            $level = $matches[1];
+            $htitle = $matches[2];
+            $slug = slugify($htitle);
+            return "<h$level id=\"$slug\" class=\"js-toc-item\">$htitle</h$level>";
+        }, $html);
+        
```

Comment:
If we're making adjustments, seems a shame to lose the `<div class="full-width">` around the images (and also lose their width/height), if that's easy to do, perhaps could have direct HTML for those bits if we cared...